vulcanai2/tests/models/test_utils.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import torch
+from vulcanai2.models.utils import (round_list,
+                                   get_one_hot,
+                                   pad,
+                                   set_tensor_device,
+                                   master_device_setter)
+
+def test_round_list():
+    """Test if the list is rounded to the desired
+    decimals"""
+    test_list = [0.83754245, 0.13249807]
+    out = round_list(test_list, decimals=3)
+    assert len(str(out[0]).split(".")[1]) == 3
+
+def test_get_one_hot():
+    """Test for get_one_hot"""
+    test_inp = np.array([0,1,2,3,4,3,2,1,0])
+    n_classes = 5
+    assert get_one_hot(test_inp).shape == (len(test_inp),
+                                           n_classes)
+
+def test_pad():
+    """Test if input_tensor is padded to the desired
+    padded shape"""
+    test_nchannels = 2
+    test_nfeatures = 50
+    padded_shape = 250
+    test_inp = torch.randn([test_nchannels, test_nfeatures])
+    padded_tensor = pad(tensor=test_inp, padded_shape=[padded_shape])
+    assert padded_tensor.shape == (test_nchannels, padded_shape)
+    assert padded_tensor.nonzero().size(0) == test_nchannels*\
+                                              test_nfeatures
+
+def test_set_tensor_device():
```

Comment:
you could probably add another pytest skip here since it deals with cuda device changes

Reply:
👍